tree-sitter-stack-graphs/src/bin/tree-sitter-stack-graphs/loader.rs

```diff
@@ -17,21 +17,18 @@ use tree_sitter_stack_graphs::loader::Loader;
 
 #[derive(Args)]
 pub struct LoaderArgs {
-    /// The TSG file to use for stack graph construction
-    #[clap(long)]
-    #[clap(name = "TSG_PATH")]
+    /// The TSG file to use for stack graph construction.
+    #[clap(long, value_name = "TSG_PATH")]
```

Comment:
Could you give me a quick explanation of these annotations? I'm pretty sure I follow what they're doing, but I'm uncertain what they are, if that makes any sense.